packages/core/build/config.js

```diff
@@ -127,27 +128,46 @@ const copyConfig = base => {
 
 const generateSWConfig = is_release => ({
     cleanupOutdatedCaches: true,
-    exclude: [
-        /CNAME$/,
-        /index\.html$/,
-        /404\.html$/,
-        /^localstorage-sync\.html$/,
-        /\.map$/,
-        /sitemap\.xml$/,
-        /robots\.txt$/,
-        /manifest\.json$/,
-        /^public\/images\/favicons\//,
-        /^favicon\.ico$/,
-        /^apple-app-site-association/,
-        /^assetlinks.json/,
-        /^.well-known\//,
-        /^account\//,
-        /^js\/smartcharts\//,
-        /^bot\//,
-        /^media\//,
-        /^trader\//,
-        /^cashier\//,
-        /^js\/core\.[a-z_]*-json\./,
+    exclude: [/^(?!css).*/],
+    runtimeCaching: [
+        {
+            urlPattern: /public\/(images|sprites)\/(?!.*favicons).*$/,
+            handler: 'CacheFirst',
+            options: {
+                cacheName: 'assets',
+                expiration: {
+                    maxEntries: 15,
+                },
+            },
+        },
+        {
+            urlPattern: ({ url }) => {
+                return url.pathname.match(/^\/js\/(?!(.*((core\.[a-z_]*-json\.)|smartcharts))).*$/);
+            },
+            handler: 'CacheFirst',
+            options: {
+                cacheName: 'core-js-files',
+            },
+        },
+        {
+            urlPattern: /(account|appstore|bot|cashier|cfd|trader)\//,
+            handler: 'CacheFirst',
+            options: {
+                cacheName: 'packages-js-files',
```

Comment:
It's not specific to js. So, can we change the name to `package-files`.

Reply:
(account|appstore|bot|cashier|cfd|trader) contains styles, images, icons etc..

Reply:
we should also cache sprites in `components` package

Reply:
sprites in components package are handled in first object of the array , where the cacheName is mentioned as assets